browser-test/src/application_navigation.test.ts

```diff
@@ -1438,5 +1438,161 @@ describe('Applicant navigation flow', () => {
     })
   })
 
+  describe('using address as visibility condition', () => {
+    const programName = 'Test program for address as visibility condition'
+    const questionAddress = 'address-test-q'
+    const questionText1 = 'text-test-q-one'
+    const questionText2 = 'text-test-q-two'
+    const screen1 = 'Screen 1'
+    const screen2 = 'Screen 2'
+    const screen3 = 'Screen 3'
+
+    beforeAll(async () => {
+      const {page, adminQuestions, adminPrograms, adminPredicates} = ctx
+      await loginAsAdmin(page)
+      await enableFeatureFlag(page, 'esri_address_correction_enabled')
```

Comment:
suggestion (maybe blocking): enable `ESRI_ADDRESS_SERVICE_AREA_VALIDATION_ENABLED` as well. I think this is working without that because they're both set to true in `application.dev.conf`

Reply:
Good catch